prospector/tools/pylint/__init__.py

```diff
@@ -4,7 +4,10 @@
 import sys
 import os
 from pylint.config import find_pylintrc
-from pylint.utils import UnknownMessage
+try:
+    from pylint.utils import UnknownMessage
+except ImportError:
+    from pylint.utils import UnknownMessageError as UnknownMessage
```

Comment:
I agree, the ```pylint.exceptions``` submodule is a better choice. I see that there is a #207 in now which implements it this way, and isolates the ```try...except``` logic. I will close this PR as #207 is the one that should be merged into master.